src/styles/theme.tsx

```diff
@@ -8,6 +8,9 @@ export const theme = createTheme({
       fontSize: '32px',
       fontWeight: 700,
     },
+    body1: {
+      margin: '10px',
```

Comment:
Margin should not be a part of the typography settings in the theme, let's remove it and if needed add in the shared component